templates/long.yaml

```diff
@@ -16,3 +16,8 @@
 
 - name: Test Project - Empty
   type: Project
+
+- name: Test Team
+  type: Team
+  can_public_join: false # true for teams which members can join without an invitation or approval
+  description: Test team description
```

Comment:
[Nit:](https://google.github.io/eng-practices/review/reviewer/standard.html#:~:text=Reviewers%20should%20always%20feel%20free,they%20could%20choose%20to%20ignore.) Can you just add a newline at the end of the file to get rid of the ⛔️ in GitHub?